pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -423,14 +423,14 @@ def on_predict_batch_end(self, trainer, pl_module, outputs, batch, batch_idx, da
         self.refresh()
 
     def _get_train_description(self, current_epoch: int) -> str:
-        train_description = f"Epoch {current_epoch}"
+        train_description = f"Epoch {current_epoch + 1}"
+        if self.trainer.max_epochs is not None and self.trainer.max_epochs < 1000:
+            train_description += f"/{self.trainer.max_epochs}"
         if len(self.validation_description) > len(train_description):
             # Padding is required to avoid flickering due of uneven lengths of "Epoch X"
             # and "Validation" Bar description
-            num_digits = len(str(current_epoch))
-            required_padding = (len(self.validation_description) - len(train_description) + 1) - num_digits
-            for _ in range(required_padding):
-                train_description += " "
+            required_padding = len(self.validation_description) - len(train_description)
+            train_description += " " * required_padding
```

Comment:
Let's use fstring padding directly

https://saralgyaan.com/posts/f-string-in-python-usage-guide/#Python_f-strings_padding

Reply:
fix this.